transmission/docker-compose.yml

```diff
@@ -7,7 +7,7 @@ services:
       APP_PORT: 9091
 
   server:
-    image: linuxserver/transmission:version-3.00-r5@sha256:f0b885f211dd6fde7b64f0d6b9e5931929cfc2331fb83d6f3171e412f22b3f87
+    image: linuxserver/transmission:4.0.3@sha256:fdd97348ad51844411a6d97e9393df8c29d13581b67d2be1fb3f0919debe2bbd
```

Comment:
We need to include the sha256 digest for the mult-arch Docker image, so that the app works for Umbrel users on both arm64 and x86(amd64) devices. 

In this case, it looks like the correct digest for 4.0.3 is `sha256:be6b4fd40e4baf685633a9ccc7974fe350115fb4b2afdf244ebc0ccdf9a33cb7`

You can get this by running the command `docker pull <image>` and seeing the digest in the output. For example:

```
$ docker pull linuxserver/transmission:4.0.3
4.0.3: Pulling from linuxserver/transmission
bc9794e5ef26: Pull complete
d2e3055e0657: Pull complete
601b52927c1d: Pull complete
6dbe19f465c4: Pull complete
29ff433cf9b3: Pull complete
49f060fe6fcf: Pull complete
Digest: sha256:be6b4fd40e4baf685633a9ccc7974fe350115fb4b2afdf244ebc0ccdf9a33cb7
Status: Downloaded newer image for linuxserver/transmission:4.0.3
docker.io/linuxserver/transmission:4.0.3
```

Reply:
Yep, looks like I got the wrong one somehow. Fixed